src/languages/docker.rs

```diff
@@ -212,6 +212,7 @@ impl LanguageImpl for Docker {
                     .args(&cmds[1..])
                     .args(hook_args.as_ref())
                     .args(batch)
+                    .check(false)
```

Comment:
Hi, @j178 I set this because I think the user needs to know the container failed info